app/views/pages/forbidden.html.erb

```diff
@@ -0,0 +1,8 @@
+<h2>Forbidden</h2>
+
+<p>
+  Your account has been suspended and has limited access.
```

Comment:
If you can be bothered updating this, it could be good to start moving pages to `I18n` too 😃

Reply:
I agree, that'd be awesome, but I think it expands the scope of the PR well beyond where I was looking to go. If it were just a matter of adding a few lines to existing i18n files, that'd be cool, but "create i18n for pages" is its own body of work.